src/cli.ts

```diff
@@ -188,6 +189,22 @@ export async function main() {
     throw new Error('--sample-size must be > 0');
   }
 
+  if (opts.measure !== 'callback' && opts.measure !== 'fcp') {
+    throw new Error(
+        `Expected --measure flag to be "callback" or "fcp" ` +
+        `but was "${opts.measure}"`);
+  }
+
+  if (opts.measure === 'fcp') {
+    for (const browser of opts.browser.split(',')) {
+      if (!fcpBrowsers.has(browser)) {
+        throw new Error(
+            `Browser ${browser} does not support the ` +
+            `first contentful paint (FCP) measurement`);
```

Comment:
Consider making this just a warning rather than an error, and mention that execution may never terminate / time out if the browser does not support FCP.

Reply:
I guess on the other hand, we only support a small number of browsers right now, and so we know which browsers are supported.

Reply:
Yeah, I think a fast failure is more friendly.